Documentation/gettingstarted/gsg_sw_demo.rst

```diff
@@ -15,7 +15,7 @@ Each deployment is identified using the Kubernetes labels (``org=empire, class=d
 and (``org=alliance, class=xwing``).
 It also includes a deathstar-service, which load-balances traffic to all pods with label (``org=empire, class=deathstar``).
 
-.. parsed-literal::
+.. code-block:: shell-session
```

Comment:
Unfortunately if we do this, then the `\ |SCM_WEB|` macro below will break, so this won't link correctly into the codebase.

Reply:
oops...makes sense.  i've revert this change.